lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -7,7 +7,14 @@
    calculator.ceremony_country == 'monaco' ||
    calculator.ceremony_country == 'france' ||
    calculator.ceremony_country == 'new-caledonia' ||
-   calculator.ceremony_country == 'wallis-and-futuna' %>
+   calculator.ceremony_country == 'wallis-and-futuna' ||
+   calculator.ceremony_country == 'french-guiana' ||
+   calculator.ceremony_country == 'french-polynesia' ||
+   calculator.ceremony_country == 'guadeloupe' ||
+   calculator.ceremony_country == 'martinique' ||
+   calculator.ceremony_country == 'mayotte' ||
+   calculator.ceremony_country == 'reunion' ||
+   calculator.ceremony_country == 'st-pierre-and-miquelon' %>
```

Comment:
As you've already spotted, this happens in a later commit titled "DRY up condition for link to consular fees".